packages/headless/doc-parser/src/actions-resolver.ts

```diff
@@ -0,0 +1,225 @@
+import {
+  ApiEntryPoint,
+  ApiInterface,
+  ApiItem,
+  ApiItemKind,
+  ApiNamespace,
+  ApiVariable,
+} from '@microsoft/api-extractor-model';
+import {DocNode, DocParamBlock} from '@microsoft/tsdoc';
+import {findApi} from './api-finder';
+import {AnyEntity, Entity, ObjEntity} from './entity';
+import {buildObjEntity} from './entity-builder';
+import {resolveInterfaceMembers} from './interface-resolver';
+import {emitAsTsDoc} from './tsdoc-emitter';
+
+interface ActionsNamespace {
+  name: string;
+  members: Action[];
+}
+
+enum ActionKind {
+  Normal = 'normalAction',
+  Analytics = 'analyticsAction',
```

Comment:
In this PR, I removed the "asyncthunk" type and decided to divide actions by "normal" and "analytics" actions.